### PR TITLE
Add tooltip component

### PR DESCRIPTION
**Pull request by zac-robinson**
Ticket: https://edozohq.atlassian.net/browse/MAPS-75

Why this is needed: Adds the tooltip wrapper

Link to Figma doc:

### CODE REVIEW

File: src/Tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Tippy from '@tippyjs/react';
+import 'tippy.js/dist/tippy.css';
+
+type TootipProps = {
```
Comment (martinmcneela) [inline src/Tooltip/Tooltip.tsx:5]: Can you change 'Tootip' to 'Tooltip'

File: src/Tooltip/Tooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  children: React.ReactElement;
+  tooltipContent: React.ReactElement;
+};
+export const Tooltip: React.FC<TootipProps> = ({ tooltipContent, children, ...rest }) => (
```
Comment (martinmcneela) [inline src/Tooltip/Tooltip.tsx:9]: would 'tooltipContent' be better ar 'content'? I'm not sure so just asking the question, the thought is, it is in the tooltip, so it doesn't need to say 'tooltip' again?

File: src/Tooltip/Tooltip.tsx [hunk 3]
```diff
@@ -2,12 +2,13 @@ import React from 'react';
...
+type TooltipProps = {
   children: React.ReactElement;
-  tooltipContent: React.ReactElement;
+  content: React.ReactElement;
+  interactive: boolean;
 };
-export const Tooltip: React.FC<TootipProps> = ({ tooltipContent, children, ...rest }) => (
-  <Tippy {...rest} content={tooltipContent}>
+export const Tooltip: React.FC<TooltipProps> = ({ interactive, content, children, ...rest }) => (
+  <Tippy interactive={interactive} {...rest} content={content}>
```
Comment (martinmcneela) [inline src/Tooltip/Tooltip.tsx:11]: Ah ok, I see what is going on now. Do you think `interactive` needs to be explicit? I think we we just needed to add ti to the story to allow it to happen.

File: package.json [hunk 4]
```diff
@@ -28,9 +28,8 @@
     "lint:css": "stylelint './src/**/*.ts'"
   },
   "dependencies": {
-    "@tippyjs/react": "^4.0.0-alpha.4",
-    "downshift": "^3.4.8",
-    "tippy.js": "^6.1.1"
```
Comment (martinmcneela) [inline package.json]: Do we not need this? We are still importing the css from it in Tooltip.jsx

Comment (zac-robinson) [reply package.json]: When I play around with it in storybook and in playground, it doesn't have any issues

Comment (zac-robinson) [reply package.json]: the original tippy is a dependency of tippyjs react anyway

**Pull request merged**